variables.tf

```diff
@@ -166,11 +188,23 @@ variable "alb_internal" {
   default     = false
 }
 
+variable "alb_vpc_id" {
+  type        = string
+  description = "The ID of ALB VPC to create ALB resources in. Defaults to the ID of ecs_vpc_id if none specified."
+  default     = ""
```

Comment:
Can we just omit this default altogether or does it have to be an empty string?

Reply:
This requires an empty string for proper conditional evaluation